### PR TITLE
[Merged by Bors] - add a note on comittee size and iterations limit

### PR DESCRIPTION
**Pull request by dshulyak**

### CODE REVIEW

File: hare3/hare.go [hunk 1]
```diff
@@ -79,6 +79,11 @@ func (cfg *Config) roundStart(round IterRound) time.Duration {
 
 func DefaultConfig() Config {
 	return Config{
+		// NOTE(talm) We aim for a 2^{-40} error probability; if the population at large has a 2/3 honest majority,
+		// we need a committee of size ~800 to guarantee this error rate (at least,
+		// this is what the Chernoff bound gives you; the actual value is a bit lower,
+		// so we can probably get away with a smaller committee).For a committee of size 400,
```
Comment (poszu) [inline hare3/hare.go]:
```suggestion
		// so we can probably get away with a smaller committee). For a committee of size 400,
```

**Review by poszu — APPROVED** [review]

**Comment by dshulyak** [comment]
bors merge

**Comment by dshulyak** [comment]
bors merge

**Review by fasmat — APPROVED** [review]

**Comment by dshulyak** [comment]
bors merge

**Pull request closed without merging**